src/snapred/backend/dao/ingredients/GroceryListItem.py

```diff
@@ -26,12 +27,14 @@ class GroceryListItem(BaseModel):
     # -- "" tells FetchGroceries to choose the loader
     loader: Literal["", "LoadGroupingDefinition", "LoadNexus", "LoadEventNexus", "LoadNexusProcessed"] = ""
 
-    # the correct combinaton of the below must be set -- neutron and grouping require a runNumber,
+    # the correct combination of the below must be set -- neutron and grouping require a runNumber,
     #   grouping additionally requires a groupingScheme
     runNumber: Optional[str]
     version: Optional[str]
     groupingScheme: Optional[str]
 
+    unit: Optional[str]
```

Comment:
Should this be `Optional[Literal[wng.Units.TOF, wng.Units.DSP]]`?

Reply:
Good idea, although I wish we were just using `Enum`.  I'll add this change.

Reply:
Regardless, I'm good with moving forward if you are.